exercises/phone-number/examples/success-standard/src/Phone.hs

```diff
@@ -1,28 +1,11 @@
-module Phone (number, areaCode, prettyPrint) where
-import Data.Char (isDigit)
+module Phone (number) where
+import Data.Char (isDigit, isLetter)
 
 number :: String -> Maybe String
 number input
+  | any isLetter input = Nothing
   | len == 10 = Just digits
   | len == 11 && head digits == '1' = Just $ tail digits
   | otherwise = Nothing
-  where digits = filter isDigit input
+  where digits = filter (\c -> isDigit c || isLetter c) input
```

Comment:
The `isLetter` case is already handled in L6, so you can keep the original `digits` definition.